Gemfile

```diff
@@ -46,8 +46,8 @@ group :development, :test do
   gem 'solr_wrapper', '~> 0.10'
   gem 'fcrepo_wrapper', '~> 0.4'
 
-  gem 'rubocop', '~> 0.43.0'
-  gem 'rubocop-rspec', '~> 1.8.0'
+  gem 'rubocop', '~> 0.43'
```

Comment:
This seems like it'll break next time rubocop is updated.  I'd prefer we handle the updates on our terms.